doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md

```diff
@@ -12,7 +12,13 @@ sort_idx: 30
 
 This guide explains how to build applications for Texas Instruments CC3200 LaunchPad (further, CC3200) based on the Kaa C SDK.
 
-# Installing necessary components for Linux (Ubuntu)
+## Installing necessary components for Linux (Ubuntu)
+
+**All steps described in this guide were tested on:**
+
+ - **Host OS:** Ubuntu 14.04 LTS Desktop 64-bit.
+ - **Device:** [TI CC3200 LaunchPad](http://www.ti.com/tool/cc3200-launchxl)
+ - **Target OS:** [TI-RTOS](http://www.ti.com/tool/ti-rtos)
```

Comment:
It is better to specify the CC3200 SDK version instead of OS. The latest SDK that I used is "CC3200 SDK v1.2.0 release". 
We actually do not use TI-RTOS.